index.js

```diff
@@ -0,0 +1,3 @@
+export function requireContract(contractName) {
```

Comment:
JavaScript in arc?  Hmm...  ;-)  Anyway, what is this needed for?  arc-js has it's own requireContract.  BTW, we should probably sync up on my work on https://github.com/daostack/arc-js/issues/17